### ISSUE TITLE
Icons alignment or responsive Size issue

### ISSUE THREAD

**Issue by yahu1031** [YazeedAlKhalaf/Flutter_Installer#10]
**Issue Reference Image**
<img src="https://cdn.discordapp.com/attachments/756903745241088011/772756384462798878/2020-11-02_03-53.png"/>

**OS**: [ParrotOS](https://www.parrotsec.org/)(Linux Distro)

**Issue in detail**

when we maximise the screen, The check/tick icons and the indicator dot/circle in stepsView has been enlarged as we show in above picture. This may be due to the responsive sizes or Wrong Icon alignment.

**Fix**

In `/lib/src/ui/widgets/step_widget.dart`
[line 59](https://github.com/YazeedAlKhalaf/Flutter_Installer/blob/master/lib/src/ui/widgets/step_widget.dart#L59)

```dart
CircleAvatar(
  maxRadius: blockSize(context)*2.5,
  minRadius: blockSize(context)*2,
  backgroundColor: Theme.of(context).scaffoldBackgroundColor,
),
```
Adding maxRadius and minRadius 
and also adding Icon size

In `/lib/src/ui/widgets/step_widget.dart`
[line 26](https://github.com/YazeedAlKhalaf/Flutter_Installer/blob/master/lib/src/ui/widgets/step_widget.dart#L26)

```drat
return Icon(
  Icons.done,
  size: blockSize(context) * 2.75,
  color: Theme.of(context).accentColor,
);
```
will fix this issue.

**Comment by YazeedAlKhalaf** [comment]
I have restricted resizing, though not the best way.
I have added a method that checks in the build method of each screen whether the window size changed or not and if changed it resizes to 1200x720.

this is the method, full code here: [https://github.com/YazeedAlKhalaf/Flutter_Installer/blob/master/lib/src/app/services/window_size_service.dart](https://github.com/YazeedAlKhalaf/Flutter_Installer/blob/master/lib/src/app/services/window_size_service.dart)
```dart
Future<void> initialize() async {
    PlatformWindow platformWindow = await _getPlatformWindow();

    if (platformWindow.screen != null) {
      if (platformWindow.screen.visibleFrame.width != width ||
          platformWindow.screen.visibleFrame.height != height) {
        _setWindowSize(platformWindow);
      }
    }
  }
```

**Comment by yahu1031** [comment]
Yeah I got it; but the thing is, it can be viewed in full size in linux(1920x1080). I have made changes depending on linux distro. I may also need to work on windows too. If you have linux just give it a shot. 😉

**Comment by YazeedAlKhalaf** [comment]
@yahu1031 
does this issue happen in this repo or on your fork?

**Comment by yahu1031** [comment]
from here and my fork both

**Comment by YazeedAlKhalaf** [comment]
I have a function that runs that makes the window stay at a specific size. If you change it, the window will resize.
Currently, I don't use native ways of doing so but I am planning on adding some native code to make the window nonresizable!
